opentracing/span.py

```diff
@@ -43,25 +43,6 @@ class Span(object):
     def __init__(self, tracer):
         self._tracer = tracer
 
-    def __enter__(self):
```

Comment:
it allows code like this:
```python
with span:
    # do stuff
```
which guarantees the span will be finished even if exception is thrown (we can add this to comments)